.github/workflows/nightly.yml

```diff
@@ -437,6 +437,26 @@ jobs:
         uses: ./.github/actions/notify-slack
         with:
           token: ${{ secrets.ACTION_MONITORING_SLACK }}
+      - name: Test Drupal
+        if: always()
+        run: |
+          php -i|grep memory_limit
+          echo memory_limit=256M > /etc/php.d/drupal.ini
+          git clone https://git.drupalcode.org/project/drupal.git drupal --depth=1
+          cd drupal
+          git rev-parse HEAD
+          php /usr/bin/composer install --no-progress --ignore-platform-reqs
+          export ASAN_OPTIONS=exitcode=139
+          export SIMPLETEST_BASE_URL=http://localhost:8888/
+          export SIMPLETEST_DB=sqlite://localhost/db/sqlite1
+          mkdir db
+          rm -f core/tests/Drupal/FunctionalTests/ExistingDrupal8StyleDatabaseConnectionInSettingsPhpTest.php
+          export DRUPAL_TYPES=Annotation,Assertion,ClassFinder,Drupal,Extension,File,Plugin,Render,tar,zip
+          php -dzend.assertions=1 vendor/bin/phpunit -c core/phpunit.xml.dist --testsuite unit,kernel --group $DRUPAL_TYPES || exit 1
```

Comment:
Is there a point to adding assertions? I'm not against it, but we if we do we should do so for all projects. Generally, we're not interested in correct behavior, just crashes/memory violations.

Reply:
The point is that lots of tests require assertions to throw so php-unit can separate errors in data-providers

Reply:
I used to pick the most important test groups and test groups, my next goal is to add a benchmark like WP and SF has